src/webapp/components/app-bar/GlassAppBar.tsx

```diff
@@ -66,10 +80,19 @@ export const GlassAppBar: React.FC = () => {
                         <IconButton aria-label="search" color="primary">
                             <LocationIcon />
                         </IconButton>
-                        <Select value={country} disableUnderline onChange={changeCountry}>
-                            <MenuItem value={1}>{i18n.t("Spain")}</MenuItem>
-                            <MenuItem value={2}>{i18n.t("France")}</MenuItem>
-                        </Select>
+                        {currentUser?.userOrgUnitsAccess && (
+                            <Select value={orgUnit} disableUnderline onChange={e => changeOrgUnit(e)}>
```

Comment:
`e => changeOrgUnit(e)` -> `changeOrgUnit`

Reply:
Addressed in https://github.com/EyeSeeTea/glass-dev/commit/c664c49dc8abcbeb0baeceac61d29b7d7275cb08